code/mob/living/carbon/human/gimmick.dm

```diff
@@ -119,6 +119,76 @@
 		bioHolder.mobAppearance.gender = "male"
 		bioHolder.mobAppearance.underwear = "boxers"
 		. = ..()
+/mob/living/carbon/human/angel
+//I have no clue if this is going to work or not
+	New()
+		..()
+		SPAWN_DBG(0)
+			src.real_name = "??????"
+			src.add_ability_holder(/datum/abilityHolder/gimmick)
+			src.nodamage = 0
```

Comment:
```suggestion
			src.nodamage = FALSE
```
Isn't `nodamage` false by default on humans, anyhow?

Reply:
Yes! That's remnant code from the jester that I changed, and (theoretically) should/can be deleted without any problems. Furthermore, the game doesn't seem to care whether or not it's set to 0 or false, and both exhibit the same behavior.